constraints.txt

```diff
@@ -297,7 +307,7 @@ pathspec==0.12.1
     # via black
 pexpect==4.9.0
     # via ipython
-platformdirs==3.10.0
+platformdirs==2.6.2
```

Comment:
is there a reason to revert patformdirs to 2.5.2?  could we changed platformdirs to 3.10.0?

Reply:
There is no reason, will revert.

Reply:
That's funny, because the CI container complains with 2.6.2.  Will message you separately about it.